specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,1999 @@
+                        "type": "string"
+                    }
+                },
+                "capabilities": {
+                    "type": "array",
+                    "description": "The custom API capabilities",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "swagger": {
+                    "type": "object",
+                    "description": "The JSON representation of the swagger"
+                },
+                "brandColor": {
+                    "type": "string",
+                    "description": "Brand color"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "The custom API desciption"
+                },
+                "displayName": {
+                    "type": "string",
+                    "description": "The display name"
+                },
+                "iconUri": {
+                    "type": "string",
+                    "description": "The icon URI"
+                },
+                "backendService": {
+                    "$ref": "#/definitions/ApiResourceBackendService"
+                },
+                "apiDefinitions": {
+                    "$ref": "#/definitions/ApiResourceDefinitions"
+                },
+                "apiType": {
+                    "$ref": "#/definitions/ApiType"
+                },
+                "wsdlDefinition": {
+                    "$ref": "#/definitions/WsdlDefinition"
+                }
+            }
+        },
+        "InsensitiveDictionaryOfJToken": {
+            "type": "object",
+            "description": "A dictionary of parameters",
+            "additionalProperties": false,
+            "allOf": [
+                {
+                    "type": "object",
+                    "additionalProperties": {}
+                }
+            ]
+        },
+        "ApiResourceBackendService": {
+            "type": "object",
+            "description": "The API backend service",
+            "additionalProperties": false,
+            "properties": {
+                "serviceUrl": {
+                    "type": "string",
+                    "description": "The service URL"
+                }
+            }
+        },
+        "ApiResourceDefinitions": {
+            "type": "object",
+            "description": "API Definitions",
+            "additionalProperties": false,
+            "properties": {
+                "originalSwaggerUrl": {
+                    "type": "string",
+                    "description": "The original swagger URL"
+                },
+                "modifiedSwaggerUrl": {
+                    "type": "string",
+                    "description": "The modified swagger URL"
+                }
+            }
+        },
+        "ApiType": {
+            "type": "integer",
+            "description": "The API type",
+            "x-enumNames": [
```

Comment:
here and below use the actual string values as enum (same as, e.g., for status)

Reply:
Changed